src/backend/dx12/src/command.rs

```diff
@@ -1243,9 +1244,12 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
         }
     }
 
-    fn bind_vertex_buffers(&mut self, vbs: pso::VertexBufferSet<Backend>) {
+    fn bind_vertex_buffers(&mut self, first_binding: u32, vbs: pso::VertexBufferSet<Backend>) {
         // Only cache the vertex buffer views as we don't know the stride (PSO).
-        for (&(buffer, offset), view) in vbs.0.iter().zip(self.vertex_buffer_views.iter_mut()) {
+        self.vertex_buffers_start_slot = first_binding;
```

Comment:
Multiple calls to bind_vertex_buffers will overwrite this. As the actual native call is deferred, we need to keep track of which vertex buffer are updated.